bin/pygrb/pycbc_pygrb_plot_chisq_veto

```diff
@@ -315,3 +319,4 @@ plu.pygrb_plotter(trigs, injs, x_label, y_label, opts,
                   snr_vals=snr_vals, conts=conts, colors=colors,
                   shade_cont_value=cont_value, vert_spike=True,
                   cmd=' '.join(sys.argv))
+                                                                                                                                                                                                                                                                                                                                                                                                                                                         321,41        Bot
```

Comment:
This looks like an accidental paste, which is causing the tests to fail. Remove.